README.md

```diff
@@ -11,7 +11,7 @@ You can read the document [about virtual machine extensions and features](https:
 | Name | Lastest Version | Description |
 |:---|:---|:---|
 | [Custom Script](./CustomScript) | 1.5 | Allow the owner of the Azure Virtual Machines to run customized scripts in the VM |
-| [DSC](./DSC) | 2.70 | Allow the owner of the Azure Virtual Machines to configure the VM using Windows PowerShell Desired State Configuration (DSC) for Linux |
+| [DSC](./DSC) | 2.71 | Allow the owner of the Azure Virtual Machines to configure the VM using Windows PowerShell Desired State Configuration (DSC) for Linux |
```

Comment:
Did you get change to work with riva and start updating the public documentation?

Reply:
Below is the public documentation that we have. It already is updated to add Oracle 6 & 7. Ubuntu 18 as supported distros.
https://docs.microsoft.com/en-us/powershell/scripting/dsc/getting-started/lnxgettingstarted?view=powershell-7.1

This PR will update the distros supported by the extension. It is available as part of Readme.